src/models/scripts/script.entity.ts

```diff
@@ -0,0 +1,32 @@
+import { Entity, Column, PrimaryGeneratedColumn, ManyToOne } from 'typeorm';
+import { User } from '../users/user.entity';
+import { Workspace } from '../workspaces/workspace.entity';
+
+@Entity({
+  name: 'scripts',
+})
+export class Script {
+  @PrimaryGeneratedColumn()
+  id: number;
+
+  @Column()
+  path: string;
+
+  @Column()
+  name: string;
+
+  @Column()
+  description: string;
+
+  @Column()
+  isPublic: boolean;
+
+  @Column()
+  createdAt: string;
+
+  @ManyToOne(() => Workspace, (workspace) => workspace.scripts)
```

Comment:
Also my mistake. Change to many to many. See scheme [here](https://www.figma.com/file/dDH2TCvH32jfYVj6e0ZHfd/pani-things?node-id=33%3A157)